### PR TITLE
APAC exceptions for membership guidelines

### PR DESCRIPTION
**Pull request by joenash**
- Added exceptions regarding Hardware lab

**Questions to resolve before merge**:

- There are some links within the guidelines to Google Drive docs. If we intend to support mainland China, we should provide alternatives.
- Several docs have named contacts, such as code of conduct, cheating incident response, etc - are these parties prepared to accept reports from APAC before the India team is in place? 
- For generic reporting, an APAC number would be good - Twilio forwarding?

### CODE REVIEW

**Comment by joenash** [comment]
Is this PR sufficient to also update the Membership Agreement, or will that need these changes ported to HelloSign?

**Comment by theycallmeswift** [comment]
LGTM

**Comment by theycallmeswift** [comment]
@joenash these changes LGTM. Needs to be ported to S3 and HelloSign. Can you work with @nquinlan on that?

**Comment by joenash** [comment]
@theycallmeswift sure thing

**Comment by joenash** [comment]
- Added exclusions for First Time Event Grant & specific sponsor swag
  - Whilst we have offered select events a first time grant paid for by the MS budget, we'll keep this offering limited and employ it as necessary. 
- @theycallmeswift to provide APAC number for CoC
- @joenash to provide @theycallmeswift pdfs for S3 upload
   - [ ] Code of conduct
   - [ ] Membership guidelines

**Comment by theycallmeswift** [comment]
We need a physical address to register the number in India. For now we'll use the email and plan to add the phone number once we have that nailed down.

File: member-event-guidelines.md [hunk 1]
```diff
@@ -92,6 +93,8 @@ benefits.
     subject to approval from GitHub. Information will be sent out 40 days
     prior to your hackathon.
 
+    - **Asia-Pacific:** A grant First Time Member Events is currently not available to Member Events in Asia-Pacific.
```
Comment (yashovardhan) [inline member-event-guidelines.md]: We need changes here

File: member-event-guidelines.md [hunk 2]
```diff
@@ -123,6 +126,8 @@ benefits.
     non-MLH Hardware into our checkout system as requested. Lab contents
     vary by event.
 
+    - **Asia-Pacific:** The hardware lab is currently not available to Member Events in Asia-Pacific.
```
Comment (yashovardhan) [inline member-event-guidelines.md]: Hardware lab is available now

File: member-event-guidelines.md [hunk 3]
```diff
@@ -157,6 +162,8 @@ benefits.
     products. Events may also receive assorted swag from MLH Season
     sponsors such as pens and t-shirts.
 
+    - **Asia-Pacific:** Select swag may not be available to Member Events in Asia-Pacific.
```
Comment (yashovardhan) [inline member-event-guidelines.md]: Can we have a discussion on this?

File: member-event-guidelines.md [hunk 4]
```diff
@@ -335,6 +342,8 @@ organizers must meet the following requirements:
     providing volunteers, our MLH rep will be freed up to help your
     organizing team and hackers with higher priority event support.
 
+    - **Asia-Pacific:** Reservation of a table for the MLH Hardware Lab is not required.
```
Comment (yashovardhan) [inline member-event-guidelines.md]: It's required as of now imo

File: member-event-guidelines.md (hunk 2 above)
Comment (theycallmeswift) [reply member-event-guidelines.md]: Correct, you should change it

File: member-event-guidelines.md (hunk 4 above)
Comment (theycallmeswift) [reply member-event-guidelines.md]: Required only if the hardware lab is available at their event

File: member-event-guidelines.md (hunk 3 above)
Comment (theycallmeswift) [reply member-event-guidelines.md]: For now we should leave this in. We can't guarantee that every single thing we provide swag wise in North America and Europe is available in APAC. Most of it is, but this phrasing appropriately sets expectations.

**Pull request closed without merging**